Lombiq.Tests.UI.Tests.UI/TestCases/WorkflowShortcutsTestCases.cs

```diff
@@ -0,0 +1,25 @@
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Helpers;
+using Lombiq.Tests.UI.Services;
+using Lombiq.Tests.UI.Shortcuts;
+using Shouldly;
+
+namespace Lombiq.Tests.UI.Tests.UI.TestCases;
+
+public static class WorkflowShortcutsTestCases
+{
+    public static Task GenerateHttpEventUrlShouldWorkAsync(
+        ExecuteTestAfterSetupAsync executeTestAfterSetupAsync, Browser browser) =>
+        executeTestAfterSetupAsync(
+            async context =>
+            {
+                await context.EnableFeatureDirectlyAsync(ShortcutsFeatureIds.Workflows);
+                await context.ExecuteRecipeDirectlyAsync("Lombiq.Tests.UI.Tests.UI.WorkflowShortcutsTests");
+                (await context.GenerateHttpEventUrlAsync(
+                    "testworkflow000000",
+                    "testhttpevent00000"))
+                    .ShouldStartWith("/workflows/Invoke?token=");
```

Comment:
imo looks cleaner and easier to comprehend. I'm not a fan of the arguments and the subsequent method call being on the same indentation level.
```suggestion
                (await context.GenerateHttpEventUrlAsync("testworkflow000000", "testhttpevent00000"))
                    .ShouldStartWith("/workflows/Invoke?token=");
```

Reply:
This was like that already, but yeah, `ShouldStartWith()` should really start one tab to the left. But the preceding line doesn't need to be broken up indeed.